src/metabase/models/revision.clj

```diff
@@ -172,7 +173,8 @@
         last-object       (t2/select-one-fn :object Revision :model (name entity) :model_id id {:order-by [[:id :desc]]})]
     ;; make sure we still have a map after calling out serialization function
     (assert (map? serialized-object))
-    (when-not (= serialized-object last-object)
+    (when-not (= (json/generate-string serialized-object)
```

Comment:
Should we add a comment here explaining why we need to compare generated strings? It's not obvious from reading the code